src/components/AppBar.tsx

```diff
@@ -124,6 +115,7 @@ const ResponsiveAppBar: React.FC = () => {
                 </Tooltip>
                 
                 <Menu
+                disableScrollLock={true}
```

Comment:
This could be just `disableScrollLock`, in theory you don't need to add `true`, as value.
The mere presence of the attribute should be enough to say, that it's value is `true`.

Reply:
Changed.